### PR TITLE
use v3 of the build service for spec v2 components

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by notlee** [comment]
I'll carry on with this when the demos endpoint has been released and the `modules` param is updated to `components` -- then I can include integration tests

File: test/integration/routes/v1-repos-(id)-versions-(id)-demos.test.js [hunk 1]
```diff
@@ -3,99 +3,16 @@
 
 const database = require('../helpers/database');
 const assert = require('proclaim');
+const { urlencoded } = require('body-parser');
 
-describe('GET /v1/repos/:repoId/versions/:versionId/demos', () => {
-	let request;
+describe.only('GET /v1/repos/:repoId/versions/:versionId/demos', () => {
```
Comment (JakeChampion) [inline test/integration/routes/v1-repos-(id)-versions-(id)-demos.test.js:8]: Stray `.only` here

```suggestion
describe('GET /v1/repos/:repoId/versions/:versionId/demos', () => {
```

File: test/integration/routes/v1-repos-(id)-versions-(id)-demos.test.js [hunk 2]
```diff
@@ -3,99 +3,16 @@
 
 const database = require('../helpers/database');
 const assert = require('proclaim');
+const { urlencoded } = require('body-parser');
```
Comment (JakeChampion) [inline test/integration/routes/v1-repos-(id)-versions-(id)-demos.test.js:6]:
```suggestion
```

File: models/bundle.js [hunk 3]
```diff
@@ -15,15 +15,30 @@ function initModel(app) {
      *
      * @param {Version} version - the Version to update a bundle for.
      * @param {string} language - the language of the bundle to update, e.g. 'css' or 'js'.
-     * @param {string} brand [null] - the brand of the bundle to update, e.g. 'internal' (optional).
+     * @param {string} brand - the brand of the bundle to update, e.g. 'internal'.
      * @return {Bundle} - bundle information for the given version, language, and brand
      */
     async function updateBundleForVersion(version, language, brand = null) {
```
Comment (JakeChampion) [inline models/bundle.js:21]: As the JSDoc has been updated to state `brand` is no longer optional, should we remove the default value here?

```suggestion
    async function updateBundleForVersion(version, language, brand) {
```

File: test/integration/models/bundle.test.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+	beforeEach(async () => {
+		await database.seed(app, 'basic');
+
+		nock('https://www.ft.com')
+			.persist()
+			.head(uri => uri.includes('bundles'))
+			.reply(function (uri) {
+				const url = new URL(this.req.options.href);
+				const { lang } = uri.match(/bundles\/(?<lang>[a-z]*)/).groups;
```
Comment (JakeChampion) [inline test/integration/models/bundle.test.js:47]: We need to update the eslint config to enable parsing of this named regex group.

named capture groups were added in ecmascript 2018 and we have eslint using an ecmascript 2017 parser

https://github.com/Financial-Times/origami-repo-data/blob/master/.eslintrc.js#L9

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**